packages/protocol/contracts/governance/Validators.sol

```diff
@@ -402,6 +402,8 @@ contract Validators is
     require(uptime <= FixidityLib.fixed1().unwrap(), "Uptime cannot be larger than one");
     uint256 numerator;
     uint256 denominator;
+    uptime = uptime + 6250000000000000000000 > FixidityLib.fixed1().unwrap() ? FixidityLib.fixed1().unwrap() : uptime + 6250000000000000000000;
```

Comment:
What is the `6250000000000000000000` value?

Reply:
It's the uptime score hit from missing 108 blocks in an epoch. You can see how it's calculated here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0029.md#implementation

Reply:
One thing that I recommend here, is to use a variable for that number and reuse it in both places (to avoid a missing 0 for example in one of those)

```
uint256 grace_period = 6250000000000000000000;
uptime = uptime + grace_period > FixidityLib.fixed1().unwrap() ? FixidityLib.fixed1().unwrap() : uptime + grace_period;
```

Reply:
Updated to use a variable! Thanks.